armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -170,8 +171,11 @@ def performThermalAxialExpansion(
         setFuel : boolean, optional
             Boolean to determine whether or not fuel blocks should have their target components set
             This is useful when target components within a fuel block need to be determined on-the-fly.
+        coldHeightsToHot: bool
+            determines if thermal expansion factors should be calculated from c.inputTemperatureInC
+            to c.temperatureInC or some other reference temperature and c.temepratureInC
```

Comment:
its more than that though. its not just reference temperature. its performing non mass conserving expansion to accommodate coldHeightsConsidered Hot

Reply:
https://github.com/terrapower/armi/pull/1342/commits/8f969cbd1bf22e10de01ad1c0fcd98e331b50d73